### PR TITLE
BZ-1896679: Moving content to the right location

### PR DESCRIPTION
**Pull request by sagidlow**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1896679
Applies to 4.6+
SME and QE ack required (@wking and @jiajliu)
Preview: [Mirroring the OpenShift Container Platform image repository](https://deploy-preview-41550--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster#update-mirror-repository_updating-restricted-network-cluster) Step 4, first bullet, first sub-bullet.

### CODE REVIEW

File: modules/update-mirror-repository.adoc [hunk 1]
```diff
@@ -98,6 +98,18 @@ $ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}
 ----
 . Mirror the version images to the mirror registry.
 ** If your mirror host does not have internet access, take the following actions:
+... If the local container registry and the cluster are connected to the mirror host, directly push the release images to the local registry and apply the config map  to the cluster by using following command:
```
Comment (wking) [inline modules/update-mirror-repository.adoc:101]: `--from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}` is `oc` reaching out to Quay to get the release payload and reaching out to the local registry to drop off that payload where the cluster can find it.  `--apply-release-image-signature` is reaching out to [either of the signature locations][1] to pull the signature, and then reaching out to the cluster to insert a signature ConfigMap.  Neither one of the external reaches (Quay or the signature locations) will work in this case, which is `If your mirror host does not have internet access...`.  I think we want to leave the `--apply-release-image-signature` content where it is today.

The bit we do want here [the *Creating the config map for image signature verification by using the oc CLI* content][2] (which would go at the end of this `If your mirror host does not have internet access...` branch, right after the `oc image mirror ...` call pushes the sneakernetted release payload up into the local registry) to push the signature from the sneakernetted `mirror/config/signature-sha256-*.yaml` up into the local cluster.

[1]: https://github.com/openshift/cluster-update-keys/blob/342eb8e2631f3218ea386479102784d5e288d868/manifests.rhel/0000_90_cluster-update-keys_configmap.yaml#L4-L5
[2]: https://github.com/openshift/openshift-docs/blob/d4762f0f626a4dddb9d7330e63a3bb6cb73f5bb5/modules/update-oc-configmap-signature-verification.adoc

File: modules/update-mirror-repository.adoc [hunk 2]
```diff
@@ -113,16 +125,3 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
```
Comment (wking) [inline modules/update-mirror-repository.adoc:127]: right after this, but before the `If the local container registry and the cluster are connected to the mirror host...` section (which we should keep below), is where [the *Creating the config map for image signature verification by using the oc CLI* content][1] steps fit in.

[1]: https://github.com/openshift/openshift-docs/blob/d4762f0f626a4dddb9d7330e63a3bb6cb73f5bb5/modules/update-oc-configmap-signature-verification.adoc

File: modules/update-mirror-repository.adoc (hunk 1 above)
Comment (jianlinliu) [reply modules/update-mirror-repository.adoc:101]: Agree with @wking, this change definitely will break the original working procedure.

File: modules/update-mirror-repository.adoc [hunk 3]
```diff
@@ -114,6 +114,19 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
...
 
+... Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] or link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)].
```
Comment (LalatenduMohanty) [inline modules/update-mirror-repository.adoc:117]: We should remove the GCS link as it wont work without the full pull spec. It  also confuses the reader. cc @jiajliu @sdodson

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: modules/update-mirror-repository.adoc [hunk 4]
```diff
@@ -114,6 +114,19 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
...
+... Use `oc` command-line interface (CLI) to log in to the cluster that you are upgrading.
+
+... Apply the mirrored release image signature config map to the connected cluster:
++
+[source,terminal]
+----
+$ oc apply -f <image_signature_file> <1>
+----
++
+<1> For `<image_signature_file>`, specify the path and name of the file, for example, `mirror/config/signature-sha256-81154f5c03294534.yaml`.
```
Comment (wking) [inline modules/update-mirror-repository.adoc:128]: nit: probably `oc apply -f ${REMOVABLE_MEDIA_PATH}/mirror/config/<image_signature_file>` or some such above, and then the `<image_signature_file>` placeholder can just talk about the final filename like `signature-sha256-81154f5c03294534.yaml`.

File: modules/update-mirror-repository.adoc (hunk 3 above)
Comment (jianlinliu) [reply modules/update-mirror-repository.adoc:117]: Agree, I suggested to drop this step some days ago in https://bugzilla.redhat.com/show_bug.cgi?id=1896679#c4

[the remark this answers is not in the document]
Comment (jianlinliu) [reply modules/update-mirror-repository.adoc:117]: https://docs.openshift.com/container-platform/4.6/updating/updating-restricted-network-cluster.html#update-configuring-image-signature already has the steps to find-out which signature digest file, right?

**Comment by sagidlow** [comment]
@jiajliu or @jianlinliu and @wking or @LalatenduMohanty could you take a look?

File: modules/update-mirror-repository.adoc [hunk 5]
```diff
@@ -114,6 +114,17 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
 
+... Use `oc` command-line interface (CLI) to log in to the cluster that you are upgrading.
```
Comment (EricPonvelle) [inline modules/update-mirror-repository.adoc:117]: More of a note on this entire section:

Shouldn't it be 4. a. i.? Currently, the second set of lists is unordered, but there are two items there.

**Review by EricPonvelle — COMMENTED** [review]
Small structural observation but /lgtm otherwise

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

File: modules/update-mirror-repository.adoc (hunk 5 above)
Comment (sagidlow) [reply modules/update-mirror-repository.adoc:117]: It uses a bulleted list instead of numbered steps because it is one of two options.

**Pull request merged**